TicketBookingFrontend/src/core/input-with-label/input-with-label.component.css

```diff
@@ -0,0 +1,4 @@
+.error-message
+{
+    color:red;font-size: small;
+}
```

Comment:
Lets move all the common css class to a single css file

Reply:
resolved